sphinx_gherkindoc/glossary.py

```diff
@@ -33,12 +33,14 @@ def tuple_len(self) -> Tuple[int, int]:
         """Get the length for each location and the number of associated steps."""
         return (len(self.locations), sum(map(len, self.locations.values())))
 
-    def __gt__(self, other) -> bool:
+    def __gt__(self, other: "GlossaryEntry") -> bool:
```

Comment:
Why is this in quotes??

Reply:
It's a [name forward reference](https://mypy.readthedocs.io/en/stable/kinds_of_types.html?highlight=circular%20reference#class-name-forward-references), required in this case because the method is part of the class definition, so= you can't use the class itself as the type reference within the class def.

Reply:
interesting, thanks!